.github/workflows/testNginxForAzureDeploy.yml

```
@@ -3,63 +3,87 @@
         uses: actions/checkout@v2
-      - name: 'AZ CLI Login'
+      - name: "AZ CLI Login"
         uses: azure/login@v1
         with:
-            client-id: ${{ secrets.AZURE_CLIENT_ID }}
-            tenant-id: ${{ secrets.AZURE_TENANT_ID }}
-            subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
-      - name: 'Update config - single file'
+          client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+      - name: "Sync NGINX Certificate to NGINX on Azure instance - single cert"
+        uses: nginxinc/nginx-for-azure-deploy-action@v0.2.0
+        with:
+          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+          resource-group-name: $TEST_RESOURCE_GROUP_NAME
+          nginx-deployment-name: $NGINX_DEPLOYMENT_NAME
+          nginx-certificate-details: '[{"certificateName": "$NGINX_CERT_NAME", "keyvaultSecret": "https://$NGINX_VAULT_NAME.vault.azure.net/secrets/$NGINX_CERT_NAME", "certificateVirtualPath": "/etc/nginx/ssl/$GITHUB_RUN_NUMBER/my-cert.crt", "keyVirtualPath": "/etc/nginx/ssl/$GITHUB_RUN_NUMBER/my-cert.key"  } ]'
+
+      - name: "Update config - single file"
         shell: bash
         run: |
           sed -i 's/000000/'"$GITHUB_RUN_NUMBER"'/g' test/configs/single/nginx.conf
           cat test/configs/single/nginx.conf
-      - name: 'Sync NGINX configuration to NGINX on Azure instance - single file'
-        uses: nginxinc/nginx-for-azure-deploy-action@v0
+      - name: "Sync NGINX configuration to NGINX on Azure instance - single file"
+        uses: nginxinc/nginx-for-azure-deploy-action@v0.2.0
         with:
           subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
           resource-group-name: $TEST_RESOURCE_GROUP_NAME
           nginx-deployment-name: $NGINX_DEPLOYMENT_NAME
           nginx-config-directory-path: test/configs/single/
           nginx-root-config-file: $NGINX_ROOT_CONFIG_FILE
           transformed-nginx-config-directory-path: $NGINX_TRANSFORMED_CONFIG_DIR_PATH
-      - name: 'Validate config update - single file'
+      - name: "Validate config update - single file"
         shell: bash
         run: |
           wget -O - -o /dev/null http://${{ secrets.NGINX_DEPLOYMENT_IP }} | jq '.request.headers."Github-Run-Id"  | test( "'"$GITHUB_RUN_NUMBER"'")'
-      - name: 'Update config - multi file'
+      - name: "Update config - multi file"
         shell: bash
         run: |
+          sed -i 's/000000/'"$GITHUB_RUN_ID"'/g' test/configs/multi/nginx.conf
+          cat test/configs/single/nginx.conf
           sed -i 's/000000/'"$GITHUB_RUN_ID"'/g' test/configs/multi/conf.d/proxy.conf
           cat test/configs/multi/conf.d/proxy.conf
-      - name: 'Sync NGINX configuration to NGINX on Azure instance - multi file'
-        uses: nginxinc/nginx-for-azure-deploy-action@v0
+      - name: "Sync NGINX configuration to NGINX on Azure instance - multi file"
```

I know you just fixed the quotes but...

```suggestion
      - name: "Sync NGINX configuration to NGINX for Azure - multi file"
```